python_ta/patches/transforms.py

```diff
@@ -4,15 +4,18 @@
 from pylint.lint import PyLinter
 
 from ..cfg.visitor import CFGVisitor
+from ..transforms.z3_visitor import Z3Visitor
 
 
-def patch_ast_transforms():
+def patch_ast_transforms(linter_config):
```

Comment:
Just pass in the `z3` value directly (a `bool`); please also add type annotations here